clue.js

```diff
@@ -0,0 +1,340 @@
+  color: '#7ab8b8',
+  image: 'assets/study.png'
+}
+
+const spa = {
+  name: 'Spa',
+  color: '#ccd1c7',
+  image: 'assets/spa.png'
+}
+
+const observatory = {
+  name: 'Observatory',
+  color: '#667f99',
+  image: 'assets/observatory.png'
+}
+
+const stables = {
+  name: 'Stables',
+  color: '#a6a659',
+  image: 'assets/stables.png'
+}
+
+const guestHouse = {
+  name: 'Guest House',
+  color: '#d6c2d6',
+  image: 'assets/guesthouse.png'
+}
+
+const trophyRoom = {
+  name: 'Trophy Room',
+  color: '#999966',
+  image: 'assets/trophyroom.png'
+}
+
+const carriageHouse = {
+  name: 'Carriage House',
+  color: '#b8cce0',
+  image: 'assets/carriagehouse.png '
+}
+
+// Arrays for suspects, weapons, and rooms.
+const suspects = [
+  mrGreen, 
+  profPlum, 
+  missScarlett, 
+  mrsPeacock, 
+  colMustard, 
+  mrsWhite
+]
+
+const weapons = [
+  rope, 
+  knife, 
+  candlestick, 
+  leadPipe, 
+  poison, 
+  horseshoe, 
+  hatPin, 
+  wrench, 
+  revolver
+]
+
+const rooms = [
+  hall, 
+  lounge, 
+  diningRoom, 
+  kitchen, 
+  ballroom, 
+  conservatory, 
+  billiardRoom, 
+  library, 
+  study, 
+  spa, 
+  observatory, 
+  stables, 
+  guestHouse, 
+  trophyRoom, 
+  carriageHouse
+]
+
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+const randomSelector = array => {
+  return array[Math.floor(Math.random() * array.length)];
+}
+
+// Object to hold The Mystery
+const mystery = {
+  killer: '',
+  weapon: '',
+  room: ''
+}
+
+// Funtions to select a random Killer, Weapon, and Room to add to Mystery Object and display card.
+const pickKiller = () => {
+  mystery.killer = randomSelector(suspects)
+
+  const killer = document.getElementById('killer-card')
+  const killerName = document.getElementById("killerTitle")
+  const killerPic = document.getElementById("killerImg")
+  const killerFullName = document.getElementById("killerName")
+  const killerAge = document.getElementById("killerAge")
+  const killerWrk = document.getElementById("killerOccupation")
+  const killerFavWeapon = document.getElementById("killerFavWeapon")
+  const killerDescrip = document.getElementById("killerDescription")
+
+  killer.style.background = mystery.killer.color
+  killerName.innerHTML = mystery.killer.title
+  killerPic.src = mystery.killer.image
+  killerFullName.innerHTML = `AKA: ${mystery.killer.firstName} ${mystery.killer.lastName}`
+  killerAge.innerHTML = `Age: ${mystery.killer.age}`
+  killerWrk.innerHTML = `Occupation: ${mystery.killer.occupation}`
+  killerFavWeapon.innerHTML = `Favourite Weapon: ${mystery.killer.favoriteWeapon}`
+  killerDescrip.innerHTML = mystery.killer.description
+}
+
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+
+  const weapon = document.getElementById("weapon-card")
+  const weaponName = document.getElementById("weaponName")
+  const weaponPic = document.getElementById('weaponImg')
+  const weaponDesc = document.getElementById("weaponDescription")
```

Comment:
Consistent `id` naming
camelCase vs. kebab-case

For ids we use camelCase, so `weaponCard`

Reply:
went through and made sure these were consistent.